openshift/minishift-addons/rhche-prerequisites/rhche-prerequisites.addon

```diff
@@ -0,0 +1,16 @@
+# Name: rhche-prerequisites
+# Description: Setup and Configure Prerequisites to run Eclipse Che (rh-che distribution)
+# Url: https://github.com/redhat-developer/rh-che
+# Required-Vars: NAMESPACE, KEYCLOAK_DOCKER_IMAGE
+# Var-Defaults: NAMESPACE=rhche, KEYCLOAK_DOCKER_IMAGE=eclipse/che-keycloak
+# OpenShift-Version: >=3.5.0
```

Comment:
@l0rd I had issues with running Che deployments on oc 3.6 some time ago. Have you tried it with oc 3.5?